README.qmd

```diff
@@ -213,17 +214,22 @@ The figure above summarises `r format(sum(n_per_hour$Trips), big.mark = ",")` tr
 We'll use the same input data to pick-out the most important flows in Spain, with a focus on longer trips for visualisation:
 
 ```{r}
-od_large = od_multi_list |>
-  map(~ .x |>
-        group_by(origen, destino) |>
-        summarise(Trips = sum(viajes), .groups = "drop") |>
-        filter(Trips > 500) |>
-        collect()
-  ) |>
-  list_rbind() |>
-  group_by(origen, destino) |>
-  summarise(Trips = sum(Trips)) |>
+od_large = od_db |>
+  group_by(id_origin, id_destination) |>
+  summarise(Trips = sum(n_trips), .groups = "drop") |>
+  filter(Trips > 500) |>
+  collect() |>
+  |>
   arrange(desc(Trips))
+# TODO: debug this:
+# Error in `collect()`:
+# ! Failed to collect lazy table.
+# Caused by error:
+# ! rapi_execute: Failed to run query
+# Error: Conversion Error: Could not convert string 'externo' to UINT16
+# LINE 3: GROUP BY id_origin, id_destination
```

Comment:
Any ideas @e-kotov ?

Reply:
should be fixed: https://github.com/Robinlovelace/spanishoddata/pull/25#issuecomment-2288078165